app/views/thredded/posts_common/_post.html.erb

```diff
@@ -17,4 +17,8 @@
   <% if current_ability.can? :edit, post.original %>
     <%= link_to 'Edit Post', edit_post_path(post.original), class: 'thredded--post--edit' %>
   <% end %>
+
+  <% if current_ability.can? :destroy, post.original %>
+    <%= link_to 'Delete Post', messageboard_topic_post_path(post.messageboard.slug, post.postable.slug, post.original), method: :delete, class: 'thredded--post--delete' %>
```

Comment:
Haven't looked into it but I will guess that `edit_post_path` is defined in the url/path/route helper module @glebm created. I'd have to dig in to verify but if that's the case then another method may need to be defined that looks like `post_path(post.original)` and expands it out to how the path helper looks like now.